nuxt.config.js

```diff
@@ -219,13 +219,13 @@ export default {
       // domain: 'https://tr.nuxtjs.org'
       // }
       // please leave this commented out until subdomain is live
-      // {
-      //   code: 'es',
-      //   iso: 'es-ES',
-      //   file: 'es-ES.js',
-      //   name: 'Español',
-      //   domain: 'https://es.nuxtjs.org'
-      // }
+      {
```

Comment:
I think we need to leave this commented at the moment, isn't it? @debs-obrien

Reply:
yes correct otherwise it will show in the footer and won't go anywhere as we haven't got the domain setup